src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -128,7 +127,8 @@
     @Deprecated
     private transient String yaml;
 
-    private List<String> yamls;
+    @Nonnull
+    private List<String> yamls = new ArrayList<>();
```

Comment:
isn't this confusing because it will be null on upgrade ?

Reply:
See https://github.com/jenkinsci/kubernetes-plugin/blob/f2f3d8a51e976e98a81b6d33faa150dc0a47bf37/src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java#L702